docs/index.html

```diff
@@ -970,6 +969,7 @@ <h1>Title Here</h1>
 </pre>
 
 
+
```

Comment:
why the extra space?

Reply:
I’ll also work on this as soon as possible